src/rock/models/finances/resolver.js

```diff
@@ -29,43 +29,28 @@ export default {
         id,
       });
     },
-    transactions: (
-      _,
-      { people, start, end, limit, cache, skip },
-      { models, person },
-    ) => {
+    transactions: (_, { people, start, end, limit, cache, skip }, { models, person }) => {
       if (!person) return null;
-      // NOTE: Not too sure what this query is trying to do
-      // it may be trying to put together transactions
-      // from all available aliases on a given user. In Holtzman
-      // this is likely broken also as the `people` object is hard-wired to
-      // a blank array (and person was returning inconsistently before also).
-      // if (person.GivingGroupId) {
-      //   console.log({ GivingGroupId: person.GivingGroupId, start, end, limit, skip, person });
-      //   return models.Transaction.findByGivingGroup(
-      //     {
-      //       id: person.GivingGroupId,
-      //       include: people,
-      //       start,
-      //       end,
-      //     },
-      //     { limit, offset: skip },
-      //     { cache },
-      //   );
-      // }
-
-      // console.log({ limit, skip, cache });
+      if (person.GivingGroupId) {
```

Comment:
This currently breaks giving history in Apollos. We will need to find another solution.